ghost/ghost/src/nestjs/admin-api.module.ts

```diff
@@ -0,0 +1,35 @@
+import {SnippetsController} from '../http/admin/controllers/snippets.controller';
+import {SnippetsService} from '../core/snippets/snippets.service';
+import {KnexSnippetsRepository} from '../db/knex/snippets.repository';
+import {Module} from '@nestjs/common';
+
+@Module({
+    controllers: [SnippetsController],
+    exports: [SnippetsService, 'SnippetsRepository'],
+    providers: [
+        {
+            provide: 'SnippetsRepository',
+            useClass: KnexSnippetsRepository
+        },
+        SnippetsService
+    ]
+})
+export class AdminAPIModule {}
+
+/*
```

Comment:
This implementation is not working or is it an example equivalent implementation but without the decorator?